zkvm/src/vm.rs

```diff
@@ -436,17 +436,15 @@ where
     }
 
     fn unblind(&mut self) -> Result<(), VMError> {
-        // Pop expression `expr`
-        let expr = self.pop_item()?.to_expression()?;
-
-        // Pop commitment `V`
-        let v_commitment = self.pop_item()?.to_data()?.to_commitment()?;
-        let v_point = v_commitment.to_point();
-
         // Pop scalar `v`
         let scalar_witness = self.pop_item()?.to_data()?.to_scalar()?;
         let v_scalar = scalar_witness.to_scalar();
```

Comment:
let's make this impl more readable by reducing number of named entities.

E.g. 

```rust
let v = self.pop_item()?.to_data()?.to_scalar()?.to_scalar();
let V = self.pop_item()?.to_data()?.to_commitment()?.to_point();
... verify V == v*B
self.push_item(Data::Opaque(V.as_bytes().to_vec()));
```

Reply:
`V` is `Copy`, so it'll be copied for the last line automatically via `memcpy` by compiler.

Reply:
Nice, thanks!